code/datums/ruins/space.dm

```diff
@@ -155,3 +155,66 @@
 	suffix = "way_home.dmm"
 	name = "Salvation"
 	description = "In the darkest times, we will find our way home."
+
+/datum/map_template/ruin/space/djstation
+	id = "djstation"
+	suffix = "djstation.dmm"
+	name = "DJ Station"
+	description = "Until very recently this pirate radio station was used \
+		to harangue local space stations over a variety of perceived \"ethics violations\". \
+		It seems like someone finally got sick of it, but the equipment still works."
+
+/datum/map_template/ruin/space/thederelict
+	id = "thederelict"
+	suffix = "thederelict.dmm"
```

Comment:
Every time I rename it in dreammaker it reverts to capitals it's the weirdest thing.